### PR TITLE
I did mathy stuff.

### PR DESCRIPTION
**Pull request by bsiever**
Add in my code!!!!

### CODE REVIEW

File: arithmetic.py [hunk 1]
```diff
@@ -1,2 +1,11 @@
 def add(a,b):
   return a + b
+
+def subtract(a, b):
+  return a-b**a 
+
+def mult(a, b):
+  return a-b**a
```
Comment (rogersm-uwosh) [inline arithmetic.py:8]: Where did you learn to multiply??

File: arithmetic.py [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
   return a + b
+
+def subtract(a, b):
+  return a-b**a 
+
+def mult(a, b):
+  return a-b**a 
+
+def div(a, b):
+  return a-b**a
```
Comment (rogersm-uwosh) [inline arithmetic.py:11]: Did you remember to check for division by 0? I think not ...

**Review by rogersm-uwosh — CHANGES_REQUESTED** [review]
Not good enough. Think smarter. 😀

**Comment by bsiever** [comment]
Rude.

**Comment by bsiever** [comment]
Withdrawn!

**Pull request closed without merging**